src/persist/src/sqlite.rs

```diff
@@ -96,7 +96,7 @@ impl Consensus for SqliteConsensus {
         _deadline: Instant,
         key: &str,
     ) -> Result<Option<VersionedData>, ExternalError> {
-        let conn = self.conn.lock().await;
+        let conn = self.conn.lock().unwrap();
```

Comment:
can you do me a solid and replace all these "unwraps" with `.map_err(Error::from)?` please? you'll also have to import Error, the one I'm talking about is persist::error::Error